test/src/test/java/org/corfudb/runtime/collections/DiskBackedCorfuClientTest.java

```diff
@@ -565,17 +565,17 @@ void disableSecondaryIndexes(
                     .withMessage("Secondary Index anotherKey is not defined.");
         }
         { // Negative test.
-            final Table<Uuid, ExampleValue, SampleSchema.ManagedResources> table1 =
-                    corfuStore.openTable(namespace, tableName,
+            final Table<Uuid, ExampleValue, SampleSchema.ManagedResources> table =
```

Comment:
Just curious why does it have to be a new table? Does the test not pass if the old table name is used?

Reply:
+1

Reply:
Take a look at `SMRObject::open()`. If the table names are the same, we'll end up using the same `CorfuTable` reference. Unfortunately, `table.resetTableData` does not work properly with disk backed tables, so this is the only option.